test/services/database/post_repository_test.dart

```diff
@@ -47,6 +54,43 @@ void main() {
       expect(actualPost.exists, false);
     });
 
+    test("Delete deletes subcollections", () async {
```

Comment:
I think the test name can be slightly improved:
```suggestion
    test("Delete method deletes subcollections", () async {
```

Reply:
8098704de70971a2826a70ea6e0b88afc73ee568